# Google Play Store Business Analytics Dashboard

## Phase 1: Getting Started with the Dataset

### Objective

The purpose of this notebook is to perform an initial inspection of the Google Play Store dataset before any cleaning or analysis begins.

The goals are to:

- Load the dataset successfully
- Understand its dimensions
- Inspect the available columns
- Identify data types
- Detect any obvious data quality issues
- Build a foundational understanding of the data

No cleaning, transformations, or business analysis will be performed in this notebook.

## Step 1: Import Required Libraries

In [2]:
import pandas as pd

## Step 2: Load the Dataset

In this step, we will load the Google Play Store dataset into a Pandas DataFrame.

It allows us to efficiently inspect, manipulate, and analyze structured data.

Here our goal is only to verify that the dataset loads successfully. No transformations or analysis will be performed yet.

In [3]:
# Load the dataset

df = pd.read_csv("../data/raw/Google-Playstore.csv")

In [4]:
# Display the first five rows

df.head()

,App Name,App Id,Category,Rating,Rating Count,Installs,Minimum Installs,Maximum Installs,Free,Price,...,Developer Website,Developer Email,Released,Last Updated,Content Rating,Privacy Policy,Ad Supported,In App Purchases,Editors Choice,Scraped Time
0,Gakondo,com.ishakwe.gakondo,Adventure,0.0,0.0,10+,10.0,15,True,0.0,...,https://beniyizibyose.tk/#/,jean21101999@gmail.com,"Feb 26, 2020","Feb 26, 2020",Everyone,https://beniyizibyose.tk/projects/,False,False,False,2021-06-15 20:19:35
1,Ampere Battery Info,com.webserveis.batteryinfo,Tools,4.4,64.0,"5,000+",5000.0,7662,True,0.0,...,https://webserveis.netlify.app/,webserveis@gmail.com,"May 21, 2020","May 06, 2021",Everyone,https://dev4phones.wordpress.com/licencia-de-uso/,True,False,False,2021-06-15 20:19:35
2,Vibook,com.doantiepvien.crm,Productivity,0.0,0.0,50+,50.0,58,True,0.0,...,NaN,vnacrewit@gmail.com,"Aug 9, 2019","Aug 19, 2019",Everyone,https://www.vietnamairlines.com/vn/en/terms-an...,False,False,False,2021-06-15 20:19:35
3,Smart City Trichy Public Service Vehicles 17UC...,cst.stJoseph.ug17ucs548,Communication,5.0,5.0,10+,10.0,19,True,0.0,...,http://www.climatesmarttech.com/,climatesmarttech2@gmail.com,"Sep 10, 2018","Oct 13, 2018",Everyone,NaN,True,False,False,2021-06-15 20:19:35
4,GROW.me,com.horodyski.grower,Tools,0.0,0.0,100+,100.0,478,True,0.0,...,http://www.horodyski.com.pl,rmilekhorodyski@gmail.com,"Feb 21, 2020","Nov 12, 2018",Everyone,http://www.horodyski.com.pl,False,False,False,2021-06-15 20:19:35


In [5]:
df.shape

(2312944, 24)

In [6]:
df.info()

<class 'pandas.DataFrame'>
RangeIndex: 2312944 entries, 0 to 2312943
Data columns (total 24 columns):
 #   Column             Dtype  
---  ------             -----  
 0   App Name           str    
 1   App Id             str    
 2   Category           str    
 3   Rating             float64
 4   Rating Count       float64
 5   Installs           str    
 6   Minimum Installs   float64
 7   Maximum Installs   int64  
 8   Free               bool   
 9   Price              float64
 10  Currency           str    
 11  Size               str    
 12  Minimum Android    str    
 13  Developer Id       str    
 14  Developer Website  str    
 15  Developer Email    str    
 16  Released           str    
 17  Last Updated       str    
 18  Content Rating     str    
 19  Privacy Policy     str    
 20  Ad Supported       bool   
 21  In App Purchases   bool   
 22  Editors Choice     bool   
 23  Scraped Time       str    
dtypes: bool(4), float64(4), int64(1), str(15)
memory usage: 361.8

# Key Observations

Based on the initial inspection of the Google Play Store dataset, the following observations were made:

1. The dataset contains **2,312,944 records** and **24 columns**, making it a large-scale dataset suitable for production-style analytics.

2. The dataset contains a mix of **string, numeric, and boolean** data types.

3. Several columns that represent dates (such as **Released**, **Last Updated**, and **Scraped Time**) are currently stored as strings and will require conversion during preprocessing.

4. The **Installs** column is stored as text (for example, "10+", "1,000+") and will require cleaning before quantitative analysis.

5. The dataset includes multiple business dimensions such as Category, Content Rating, and Minimum Android that can be used for segmentation and comparison.

6. Monetization-related information is available through columns such as Free, Price, Ad Supported, and In App Purchases, enabling analysis of different business models.

7. Developer-related attributes such as Developer Id, Developer Website, and Developer Email can support ecosystem-level analysis.

8. Some columns are expected to contain missing values (for example, Developer Website and Privacy Policy), which will be investigated during the Data Quality Assessment phase.

9. The dataset appears comprehensive enough to answer business questions related to marketplace performance, user satisfaction, monetization, platform compatibility, and developer activity.

10. No data cleaning or transformations have been performed at this stage. The purpose of this notebook was only to inspect and understand the dataset before analysis begins.